helm/idol-mmap/README.md

```diff
@@ -0,0 +1,15 @@
+# idol-mmap
+
+Provides a deployment of a IDOL MMAP (Media Management and Analysis Platform).
+
+## OpenShift Deployment
+
+In order to comply with constraints OpenShift places on containers, the Container and Pod security contexts of the Bitnami PostgreSQL sub chart need to be disabled.
+```
+postgresql:
+  primary:
+    containerSecurityContext:
+      enabled: false
+    podSecurityContext:
+      enabled: false
```

Comment:
I would add these explicitly into the values.yaml file with defaults set to 'true' so it's easier to see what/where to change